desktop/apps/auction2/utils/artwork.js

```diff
@@ -0,0 +1,9 @@
+import _ from 'underscore'
+
+// Logic borrowed from the artwork Backbone model
+export function titleAndYear(title, date) {
+  return _.compact([
+    title && title.length > 0 ? `<em>${title}</em>` : '',
```

Comment:
If a value comes back undefined then compact it should filter it out since `undefined` and `''` are both considered falsy:

```javascript
return _.compact([ title && `<em>${title}</em>`, date])
```